src/pages/Home.vue

```diff
@@ -142,6 +145,18 @@ export default {
   methods: {
     ...mapActions("plioItems", ["purgeAllPlios"]),
     ...mapActions("sync", ["startLoading", "stopLoading"]),
+    async sortByNumberOfViewers(sortOrder) {
+      // invoked when the user clicks the sort icon next to "number of viewers" column
+
+      // remove any existing value of "unique_viewers" field. New value will be pushed below
+      this.sortByFields = this.sortByFields.filter((value) => {
+        return !value.includes("unique_viewers");
+      });
+
```

Comment:
at any point we are sorting by only one thing, no? Or are you saying successive sorting is also applied?

Reply:
no multiple sorting - remove lists.